sentry/src/main/java/io/sentry/NoOpSpan.java

```diff
@@ -0,0 +1,63 @@
+package io.sentry;
+
+import io.sentry.protocol.SentryId;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public final class NoOpSpan implements ISpan {
+
+  private static final NoOpSpan instance = new NoOpSpan();
+
+  private NoOpSpan() {}
+
+  public static NoOpSpan getInstance() {
+    return instance;
+  }
+
+  @Override
+  public @NotNull ISpan startChild(final @NotNull String operation) {
+    return new NoOpSpan();
```

Comment:
this one and other methods could return `instance` directly, right

Reply:
Yeah the goal is to reduce the instrumentation cost so makes sense to return the singleton. Otherwise we'd be better off adding a new Span so we can have an `id` laying around at least.